xt_training/utils/functional.py

```diff
@@ -9,14 +9,12 @@
 from xt_training.utils import _import_config, Tee
 
 
-def train_exit(config, runner, save_dir):
-    test_loaders = getattr(config, 'test_loaders', None)
-
+def train_exit(test_loaders, model, runner, save_dir):
```

Comment:
FYI, this change will break msg. These are the only arguments used by the default exit function, but a lot of the time you will want other things from the config, like the eval_metrics.

Reply:
I would prefer we didn't change this, because it will make it impossible to do what we need to. One thing that is really useful is being able save side-effect only metrics, like ROC curve, on exit.

Reply:
Why would you not be able to do that here? It still has access to runner. I will add backwards compatability though

Reply:
Good point, didn't think of that. In that case I'm fine with it. Don't stress too much about the backward compatibility, if you just add a `**kwargs` arg at the end, I can handle it on the MSG repo side

Reply:
Then we can remove the **kwargs later on when no one will be on an old version